web_src/less/_base.less

```diff
@@ -1565,20 +1578,8 @@ a.ui.label:hover {
     border-bottom: 1px solid var(--color-secondary);
   }
 }
-.code-view {
-  overflow: auto;
-  overflow-x: auto;
-  overflow-y: hidden;
-
-  &.has-context-menu {
-    overflow: visible;
-    overflow-x: visible;
-    overflow-y: visible;
-  }
-
-  table {
-    width: 100%;
-  }
```

Comment:
This might have side-effects, but I didn't notice any obvious ones at last. It's necessary to get the button to appear outside the box.

Reply:
`has-context-menu` is related to diff, will check that later, that class can likely be removed as well.

Reply:
The overflow stuff was added in https://github.com/go-gitea/gitea/commit/a21a7d438b90986da7757c707b07b733b0233548 and I can't remember why, so let's remove it. It may have been related to line wrapping which wasn't in back then.